estate/models/building.py

```diff
@@ -0,0 +1,83 @@
+from odoo import models, fields, api
+from odoo.exceptions import UserError
+from datetime import timedelta
+
+
+class Building(models.Model):
+    _name = "estate.buildings"
```

Comment:
Good job on your refactoring but still some small consistency issues like here (and the file names)
```suggestion
    _name = 'estate.building'
```